lib/open_food_network/feature_toggle.rb

```diff
@@ -1,21 +1,78 @@
 module OpenFoodNetwork
+  # This feature toggles implementation provides two mechanisms to conditionally enable features.
+  #
+  # You can provide an ENV var with the prefix `OFN_FEATURE_` and query it using the
+  # `ApplicationHelper#feature?` helper method. For instance, providing the ENV var
+  # `OFN_FEATURE_NEW_SHINNY_FEATURE` you could then query it from view as follows:
+  #
+  #   - if feature? :new_shinny_feature
+  #     = render "new_shinny_feature"
+  #
+  # Alternatively, you can choose which users have the feature toggled on. To do that you need to
+  # register the feature and its users from an initializer like:
+  #
+  #   require 'open_food_network/feature_toggle'
+  #   OpenFoodNetwork::FeatureToggle.enable(:new_shinny_feature, ['ofn@example.com'])
+  #
+  # It's although advisable to read the emails list from an ENV Var.
```

Comment:
I am not sure I understand this, maybe you can add an example to the comment?

Reply:
yeah, it's not very clear. My point is that `['ofn@example.com']` should come from an ENV var we provision with ofn-install such as:

```ruby
OpenFoodNetwork::FeatureToggle.enable(:new_shinny_feature, OFN_PRODUCT_TEAM_EMAILS + OFN_DEV_TEAM_EMAILS)
```